ui/app/components/secret-engine/configure-create-edit.hbs

```diff
@@ -3,34 +3,51 @@
   SPDX-License-Identifier: BUSL-1.1
 ~}}
 
-<form {{on "submit" (perform this.submitForm)}} aria-label="configure azure credentials" data-test-configure-form>
+<form
+  {{on "submit" (perform this.submitForm)}}
+  aria-label={{concat "configure" @type " credentials"}}
+  data-test-configure-form
+>
   <NamespaceReminder @mode="save" @noun="configuration" />
   <MessageError @errorMessage={{this.errorMessage}} />
+  {{#if (eq @type "aws")}}
+    <div class="box is-fullwidth is-shadowless is-marginless">
+      <p class="has-text-grey-dark">
+        Note: the client uses the official AWS SDK and will use the specified credentials, environment credentials, shared
+        file credentials, or IAM role/ECS task credentials in that order.
+      </p>
+    </div>
+    <h2 class="title is-5 has-bottom-padding-s has-top-margin-l" data-test-access-title>
+      Access to AWS
+    </h2>
+  {{/if}}
   <div class="box is-fullwidth is-sideless">
-    {{! accessType can be "azure" or "wif" - since WIF is an enterprise only feature we default to "azure" for community users and only display those related form fields. }}
     {{#if this.version.isEnterprise}}
```

Comment:
The layout logic changed some (got easier IMO). Because _only_ WIF engines use this component we don't have to do any checks for WIF engines. Instead the logic we're concerned with is:
- Is this an enterprise version? yes, allow users to change the access type
- What accessType is currently selected? If it's "account" then show non-wif fields, otherwise show wif fields.
- Is there a second config model? Yes, then show that. Second config models are not included in accessType or enterprise logic, they should just show if they exist.